spamhandling.py

```diff
@@ -130,9 +127,18 @@ def handle_spam(post, reasons, why):
         log('debug', GlobalVars.parser.unescape(s).encode('ascii', errors='replace'))
         datahandling.append_to_latest_questions(post.post_site, post.post_id, post.title)
 
-        message = prefix_ms + s
+        for reason_count in range(5, 2, -1):  # Try 5 reasons, then 4, then 3
+            reason = ", ".join(reasons[:reason_count])
+            if len(reasons) > reason_count:
+                reason += ", +{} more".format(len(reasons) - reason_count)
+            reason = reason[:1].upper() + reason[1:]  # reason is capitalised, unlike the entries of reasons list
```

Comment:
What's this for?

Reply:
Trying to reduce displayed reason count one by one until the message fits.

Reply:
Not the loop as a whole - just line 134. What's that do? What's it produce?

Reply:
The first letter would be capitalized by that line.